### ISSUE TITLE
Error when trying to duplicate a decklist

### ISSUE THREAD

**Issue by netloaf** [Null-Signal-Games/netrunnerdb#155]
In the "My Decks" section, when I select a deck in the list and hit the yellow Duplicate button, I'm thrown to a "500 Internal Server Error", with a 503 Service Unavailable at the top. It does this for any deck that I attempt to hit the duplicate button on.

**Comment by lpoulter1** [comment]
I am also seeing this.

**Comment by plural** [comment]
hmm.  i made a change to part of the deck copying code to fix #71 .  I just tried copying a few decks on my own account with various MWL settings and it seemed to be fine, though.  Any other details you have about those decks?  Anything interesting/unusual about them?  I don't have access to the netrunnerdb.com logs so I can't see the details of these particular failures.

**Comment by netloaf** [comment]
Nothing special about the decks, but I've got a bit more info. It seems to only affect decks, and as far as I see so far, it seems random which. I've tried just picking random saved decks to duplicate to test, and I don't see a pattern in the ones that work. I took one of my older decks, pre new MWL and such, and it failed to duplicate, giving me the error from before. I took out all of the cards, one by one, saving and trying to duplicate the deck with a removed card each time, until the deck was empty, and it still fails. However, I made a new deck, using the same ID, left it empty, saved it, and it let me duplicate that one. So, I'm not sure that it cares about cards currently in the deck, but I can't rule out history.

**Comment by linduxed** [comment]
Confirming that this is an issue.

**Comment by plural** [comment]
@Alsciende are you able to take a peek at any of the logs for any pointers to root cause here?

For those of you with the issue, would you be able to share particular decklists (feel free to do it privately instead of putting it in here if you need to) with a bit more discrete reproduction instructions?

If you re-create your deck states prior to the error on duplication, does the error repeat itself based on the same deck contents?

**Comment by seeds34** [comment]
Not a lot of information but might help, I've tried duplicating 2 decks.

- My own deck which duplicated fine
- A deck (Spatula City (4-1 Belgian Nats)) I had copied using the "Copy into my decks" function fails (500 error)

So I've tried some other decks:
- I tried two more of my own decks. Both duplicated fine.
- I copied the current Deck of the week (Philippine Nationals Champion Biotech - Heller's Brew) into my Decks and then duplicated, got 500 error.
- I copied a deck form deck lists (DÄMÄTSCH V1.2), same error again

I tried a few other tests:

- On Spatula City (4-1 Belgian Nats) I edited the deck and changed MWL to 2.0 (from Casual Play). After saving I duplicated and it worked
- On DÄMÄTSCH V1.2 I edited the deck and added 3 Globe Food Initiative (making the deck illegal). After saving I tried to duplicate and got a 500 error

Hope this information might help.
